### DISCUSSION TITLE
HeadlessMC as part (dependency) of a Java App

### DISCUSSION THREAD

**Discussion by vanvanni** [headlesshq/headlessmc#84] [Q&A]
Hi,

I saw this project and wondered if it is possible to make this a part of a Java app. So it's more like a HeadlessMC-API.

**Comment by 3arthqu4ke** [comment]
What do you mean with Java App? An Android App? Or just an application? I host artifacts on 'https://3arthqu4ke.github.io/maven', you can just include them like [this](https://github.com/3arthqu4ke/HMC-Specifics/blob/fab5f4a05414f22e39abfca5ce8fccc9d86af9a2/build.gradle#L32).

**Reply by vanvanni** [reply]
Asin a Java Console Application on my pc
